### DISCUSSION TITLE
Medic bot - wrong priorities and treatments

### DISCUSSION THREAD

**Discussion by Arctomachine** [FakeFishGames/Barotrauma#14188] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Medic bots in current version behave strangely.
They prefer treating minor wounds on characters far away when heavily wounded characters (including player) are right in front of them.
Moreover, they prefer using morphine for every wound type, even for bleeding. Only much later they start applying bandages, often when target either hits overdose or faints from blood loss.

### Reproduction steps

Not exactly sure how to reproduce it reliably.
I encountered it when hammerhead spawns got inside and several bots hurt each other with different melee and ranged weapons in process.
Probably should inflict different types of wounds on multiple characters at the same time

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
There are enough correct supplies when wrong treatment happens
```

**Comment by Arctomachine** [comment]
More info on wrong treatment. When character has infected wounds, but there are no antibiotic glues, medic applies saline to treat it.

**Comment by SomeRandomNoobKekeke** [comment]
Bots also often prefer to treat blood loss with alien blood instead of saline or blood packs
And they can't treat bleedeng on multiple limbs, they will just bandage one and die
And i think bots prioritize getting drunk instead of healing infected wounds

I don't trust them at all, so i just tell them to ignore medical cabinet and treat them myself with my captains  5 med skill

Would be nice if someone make some testing framework for that, but i'm a little too busy now, mb i could do it in a week or so

**Reply by Arctomachine** [reply]
At least ethanol is one of correct treatments for infected wounds, it appears in suggested list in health window
